<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Normals</class>
 <widget class="QDockWidget" name="Normals">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>249</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>249</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
  </property>
  <property name="windowTitle">
   <string>Normals</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ViewPoint:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="check_origin">
           <property name="text">
            <string>Origin</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="check_center">
           <property name="text">
            <string>Center</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="check_max">
           <property name="text">
            <string>Max</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="rbtn_k">
           <property name="text">
            <string>K.search: </string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_k">
           <property name="prefix">
            <string/>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="rbtn_r">
           <property name="text">
            <string>R.search: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dspin_r">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="maximum">
            <double>9999.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="check_reverse">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/res/icon/reverse.svg</normaloff>:/res/icon/reverse.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spin_level">
           <property name="prefix">
            <string>Level: </string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dspin_scale">
           <property name="prefix">
            <string>Scale: </string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>999.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="check_refresh">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/res/icon/real-time.svg</normaloff>:/res/icon/real-time.svg</iconset>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_preview">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Preview</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/res/icon/preview.svg</normaloff>:/res/icon/preview.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">border: 2px solid lightgray;
</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_add">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/res/icon/add.svg</normaloff>:/res/icon/add.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_apply">
           <property name="text">
            <string>Apply</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/res/icon/apply1.svg</normaloff>:/res/icon/apply1.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_reset">
           <property name="text">
            <string>Reset</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/res/icon/reset.svg</normaloff>:/res/icon/reset.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
